### DISCUSSION TITLE
Github Education - personal information

### DISCUSSION THREAD

**Discussion by antoinemcx** [community/community#48890] [GitHub Education] [answered]
Hi, I received a mail saying that I have to complete my profile with "full name as it appears in [the] image and a short bio".
So my question is : do I really have to put my name on my public profile ? So everyone will see it ?

EDIT : Alright so now I have put my full real name public (what a shame by the way, thanks github) but I'm still getting issues as you can see on this picture : 
![Capture](https://user-images.githubusercontent.com/47474006/222164367-baa8a62b-aeb7-485b-a6ae-17c8a850959f.JPG)

**Comment by antoinemcx** [comment, marked as answer]
Finally it's good......

**Comment by ShadowVik** [comment]


**Reply by antoinemcx** [reply]
I personally just had to log out and reloggin
